packages/api-plugin-fulfillment/src/schemas/schema.graphql

```diff
@@ -0,0 +1,414 @@
+}
+
+
+"Defines a fulfillment method in general."
+type FulfillmentMethodObj {
+  "The fulfillment method ID"
+  _id: ID!
+
+  "The cost of this fulfillment method to the shop, if you track this"
+  cost: Float
+
+  """
+  The fulfillment types for which this method may be used. For example, `shipping` or `digital`.
+  """
+  fulfillmentTypes: [FulfillmentType]
+
+  "The group to which this method belongs"
+  group: String!
+
+  "A fixed price to charge for handling costs when this fulfillment method is selected for an order"
+  handling: Float!
+
+  "Include this as a fulfillment option shown to shoppers during checkout?"
+  enabled: Boolean!
+
+  "The name of this method, for display in the user interface"
+  label: String!
+
+  "The name of this method, a unique identifier"
+  name: String!
+
+  "The common-name for this method, to group all variants of the same method"
+  fulfillmentMethod: String
+
+  "A fixed price to charge for fulfillment costs when this fulfillment method is selected for an order"
+  rate: Float!
+}
+
+"Provider info"
+type ProviderInfo{
+  "Name of the provider"
+  name: String!
+  
+  "Label of the provider"
+  label: String!
+
+  "Flag defining enabled/disabled status"
+  enabled: Boolean
+}
+"Fulfillment type root object"
+type FulfillmentTypeObj implements Node {
+  "The Fulfillment Type ID"
+  _id: ID!
+
+  "The user provided name of the fulfillment type"
+  name: String!
+
+  "Shop ID"
+  shopId: ID!
+
+  "ProviderInfo"
+  provider: ProviderInfo
+
+  "Fulfillment type"
+  fulfillmentType: String
+
+  "Fulfillment methods"
+  methods: [FulfillmentMethodObj]
+}
+
+"Custom Fulfillment method object "
+type CustomFulfillmentMethodObj implements Node {
+  "The Fulfillment Type ID"
+  fulfillmentTypeId: ID!
+
+  "Shop ID"
+  shopId: ID!
+
+  "The fulfillment method ID"
+  _id: ID!
+
+  "The cost of this fulfillment method to the shop, if you track this"
+  cost: Float
+
+  """
+  The fulfillment types for which this method may be used. For example, `shipping` or `digital`.
+  """
+  fulfillmentTypes: [FulfillmentType]
+
+  "The group to which this method belongs"
+  group: String!
+
+  "A fixed price to charge for handling costs when this fulfillment method is selected for an order"
+  handling: Float!
+
+  "Include this as a fulfillment option shown to shoppers during checkout?"
+  enabled: Boolean!
+
+  "The name of this method, for display in the user interface"
+  label: String!
+
+  "The name of this method, a unique identifier"
+  name: String!
+
+  "The common-name for this method, to group all variants of the same method"
+  fulfillmentMethod: String
+
+  "A fixed price to charge for fulfillment costs when this fulfillment method is selected for an order"
+  rate: Float!
+}
+
+"Input needed to select a fulfillment option for a single fulfillment group on a cart"
+input updateFulfillmentTypeInputInfo {
+  "The shop to which this group belongs"
+  shopId: ID!
+
+  "The group to update"
+  fulfillmentGroupId: ID!
```

Comment:
Am i getting things wrong or this should be `fulfillmentTypeId` rather than `group`. If so, also address the comments of the input.

Reply:
Updated the schema elements and the comments
'Groups' are now retained only for the specific field which tracks the group (ground/free...) from the old plugin.
Update will be pushed soon